internal/tweets/home.go

```diff
@@ -0,0 +1,50 @@
+package tweets
+
+import (
+	"Twitter_like_application/internal/database/pg"
+	"Twitter_like_application/internal/services"
+	"encoding/json"
+	"github.com/lib/pq"
+	"log"
+	"net/http"
+)
+
+func Home(w http.ResponseWriter, r *http.Request) {
+	userID := r.Context().Value("userID").(int)
+	followingQuery := "SELECT following FROM follower WHERE $1"
+	rows, err := pg.DB.Query(followingQuery, userID)
+	if err != nil {
+		services.ReturnErr(w, err.Error(), http.StatusInternalServerError)
+	}
+	defer rows.Close()
+	var followingID []int
+	for rows.Next() {
+		var following int
+		if err = rows.Scan(&following); err != nil {
+			log.Println("Error scanning following", err)
+			services.ReturnErr(w, err.Error(), http.StatusInternalServerError)
+		}
+		followingID = append(followingID, following)
```

Comment:
this is wrong, arguments are backwards

Reply:
oh, right, not backwards, but misleadingly named, `followingID` implies that it's a single ID, not a list